WikiTreeAPI.js

```diff
@@ -9,6 +9,57 @@
 // Put our functions into a "WikiTreeAPI" namespace.
 window.WikiTreeAPI = window.WikiTreeAPI || {};
 
+/**
+ * Serializes WikiTree fuzzy date
+ * @param  {object}  person Person object received from WikiTree API
+ * @param  {string}  fieldName Name of the fuzzy date to be serialized, possible values: `BirthDate`, `DeathDate`
+ * @param  {boolean} [monthNames=true] Will use month names, instead of month numbers
+ * @param  {boolean} [shortened=true] Will shorten month names (if used) to 3-letter abbrevs
+ * @param  {boolean} [withCertainty=true] Will add certainty if loaded
+ * @return {string} Serialized date
+ */
+window.wtDate = function (person, fieldName, monthNames = true, shortened = true, withCertainty = true) {
```

Comment:
I've made global utility function for serializing fuzzy dates because it's used everyvery and therefore really necessary, @bcaseyrls can you please check it?

I'm not english native speaker and I'm not really sure about correct date format 🤔. What is correct? `Jun 15, 1970` or `15 Jun 1970`? (or is it rather regional issue - US vs UK? 🤔 )

I'm planning to make something similar for names.

Reply:
by @Clarke-11007 (moved here for clarity - please use those threads)
> In my Fan Chart app, I plan to give date format options , can those be an
optional parameter ?
>
>The ones I’d use are
>
>YYYY
>DDMMMYYYY
>MMMDD,YYYY
>YYYY-MM-DD
>
>for
>
>2022
>31 Oct 2022
>Oct 31, 2022
>2022-10-31
>respectively

Yes, maybe the formatting string is the correct way 🤔

Reply:
here some formatting rules from Microsoft: https://learn.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings

Reply:
added [Parameterized complete name serialization](https://github.com/wikitree/wikitree-dynamic-tree/pull/60/commits/63fa008bc57ef93f379ffd75bb76a74bf928bebb)